ComposeSamples/ComposeGallery/src/main/java/com/microsoft/device/display/samples/composegallery/HomePage.kt

```diff
@@ -117,62 +148,54 @@ private fun ShowListColumn(models: List<ImageModel>, modifier: Modifier) {
                             .wrapContentSize(Alignment.Center),
                         style = TextStyle(
                             fontSize = 20.sp,
-                            fontWeight = FontWeight.Bold
+                            fontWeight = FontWeight.Bold,
+                            color = MaterialTheme.colors.onSurface,
                         )
                     )
                     BasicText(
                         text = item.title,
                         modifier = Modifier
                             .fillMaxHeight()
-                            .wrapContentSize(Alignment.Center)
+                            .wrapContentSize(Alignment.Center),
+                        style = TextStyle(
+                            color = MaterialTheme.colors.onSurface,
+                        )
                     )
                 }
             }
-            Divider(color = Color.LightGray)
+
+            Divider(color = MaterialTheme.colors.onSurface)
         }
     }
 }
 
 @Composable
-fun ShowDetailWithList(models: List<ImageModel>) {
-    val imageSelectionLiveData = appStateViewModel.imageSelectionLiveData
-    val selectedIndex = imageSelectionLiveData.observeAsState(initial = 0).value
+fun ShowDetail(models: List<ImageModel>, selectedIndex: Int) {
     val selectedImageModel = models[selectedIndex]
 
-    Row(
+    Column(
```

Comment:
Actually, we can get rid of this `Column`. One `Column` inside `Crossfade` should be enough.